code/modules/mob/living/simple_animal/hostile/megafauna/swarmer.dm

```diff
@@ -262,16 +260,6 @@ GLOBAL_LIST_INIT(AISwarmerCapsByType, list(/mob/living/simple_animal/hostile/swa
 	check_friendly_fire = TRUE //you're supposed to protect the resource swarmers, you poop
 	retreat_distance = 3
 	minimum_distance = 3
-	login_text_dump = {"
```

Comment:
then return login_text_dump.
as *I* said above, there's precedent for admins doing this.